manifests/d/DuckStudio/GitHubLabelsManager/1.1/DuckStudio.GitHubLabelsManager.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: DuckStudio.GitHubLabelsManager
+PackageVersion: "1.1"
+PackageLocale: en-US
+Publisher: DuckStudio
+PublisherUrl: https://duckduckstudio.github.io/yazicbs.github.io/
+PublisherSupportUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/issues
+# PrivacyUrl:
+Author: 鸭鸭「カモ」
+PackageName: GitHub Labels Manager
+PackageUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager
+License: GNU General Public License v3.0
+LicenseUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/blob/main/LICENSE
+Copyright: Copyright (c) DuckStudio 鸭鸭「カモ」
+# CopyrightUrl:
+ShortDescription: A GitHub Labels Manager
+Description: Tools that automatically help you copy repo labels, get repo labels, and empty existing labels.
+Moniker: glm
+Tags:
+- devtools
+- labels
+- github-labels
+ReleaseNotes: Initial function implementation.
+ReleaseNotesUrl: https://github.com/DuckDuckStudio/Chinese_git/releases/tag/v1.1
+# PurchaseUrl:
+# InstallationNotes:
+# Documentations:
+ManifestType: defaultLocale
```

Comment:
Here you would use `locale` value, because you can only have a single default locale.

```
ManifestType: locale
```